changelog/8575.bugfix.md

```diff
@@ -0,0 +1 @@
+Check if port is int and raise exception in file pika.py
```

Comment:
Nitpicking now:

- I think you used the PR id in the changelog, you should use the issue id which I think is 8309.
- I would make the text a bit more descriptive : "Added check in `PikaEventBroker` constructor: if port cannot be cast to integer, raise `RasaException`."